From ab6e618a65963bd889bd92d4fb9ce16bcf3bab4d Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Tue, 17 Apr 2018 00:43:31 +0000
Subject: [PATCH] TRITON-328 Update NAPI to restify-warden@1.1.0 Reviewed by:
 Nick Zivkovic <nick.zivkovic@joyent.com> Approved by: Nick Zivkovic
 <nick.zivkovic@joyent.com>

---
 lib/util/validate.js | 8 ++++----
 package.json         | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/lib/util/validate.js b/lib/util/validate.js
index 80fc585..b3f8300 100644
--- a/lib/util/validate.js
+++ b/lib/util/validate.js
@@ -345,7 +345,7 @@ function validateVxlanID(_, name, vlan_id, callback) {
 module.exports = {
     bool: validateBoolean,
     enum: validateEnum,
-    fieldsArray: warden.fieldsArray,
+    fieldsArray: warden.arrayOfFields,
     IP: validateIP,
     IPv4: validateIPv4,
     ipArray: validateIParray,
@@ -363,10 +363,10 @@ module.exports = {
     uuidPrefix: warden.uuidPrefix,
     string: warden.string,
     stringOrArray: warden.stringOrArray,
-    stringArray: warden.stringArray,
+    stringArray: warden.arrayOfString,
     subnet: validateSubnet,
-    UUID: warden.UUID,
-    UUIDarray: warden.UUIDarray,
+    UUID: warden.uuid,
+    UUIDarray: warden.arrayOfUuid,
     VLAN: validateVLAN,
     VxLAN: validateVxlanID
 };
diff --git a/package.json b/package.json
index ea1e7b6..832ad6a 100644
--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "node-uuid": "1.4.7",
     "portolan-moray": "git+https://github.com/joyent/sdc-portolan-moray.git#7e2c4ca",
     "restify": "4.1.1",
-    "restify-warden": "0.2.0",
+    "restify-warden": "1.1.0",
     "sdc-clients": "9.2.0",
     "tape": "4.5.1",
     "trace-event": "1.3.0",
-- 
2.21.0

